src/views/bls12381.rs

```diff
@@ -623,6 +657,8 @@ impl<'a> ConvView for View<'a> {
             self.mk.comment.clone(),
         )
         .map_err(|e| ConversionsError::Ssh(e.into()))?)
+        .map_err(|e| ConversionsError::SshKey(e))?)
+        .map_err(ConversionsError::SshKey)?)
```

Comment:
same

Reply:
Fixed as of https://github.com/cryptidtech/multikey/pull/2/commits/f479bd484b9c377a0a44684ec0b4160d04db1b71